### PR TITLE
Simpliefied views and added basic bootstrap without any customization yet

### PR DESCRIPTION
**Pull request by patrikvalentiny**

### CODE REVIEW

File: src/main/java/com/movie_collection/gui/controllers/CreateMovieController.java [hunk 1]
```diff
@@ -69,12 +68,7 @@ private void selectFileChooser(ActionEvent actionEvent) {
 
         File selectedMovieFile = chooseFile.showOpenDialog(new Stage());
         if (selectedMovieFile != null) {
-            path.setText(selectedMovieFile.toURI().toString());
-            var media = new Media(selectedMovieFile.toURI().toString().replace("\\", "/"));
-            var duration = (int) media.getDuration().toSeconds();
-            durationField.setText(duration / 60 + ":" + duration % 60);
-            if (movieName.getText().isBlank())
-                movieName.setText(selectedMovieFile.getName());
+            path.setText(selectedMovieFile.toURI().toString().substring(6));
```
Comment (TomassSimko) [inline src/main/java/com/movie_collection/gui/controllers/CreateMovieController.java:71]: Thanks for clean up 🙏

File: src/main/java/com/movie_collection/gui/controllers/MovieController.java [hunk 2]
```diff
@@ -110,11 +108,12 @@ private void fillDescriptionWithAPIData(MovieDTO movieDTO, Movie selectedMovie)
         desPlot.setText(movieDTO.Plot);
         desRunTime.setText(movieDTO.Runtime);
         desCast.setText(movieDTO.imdbRating);
-        descrIReleased1.setText(movieDTO.Released);
+        desReleased.setText(movieDTO.Released);
         desImdbRating.setText(movieDTO.imdbRating);
         desDirector.setText(movieDTO.Director);
         descrMovieTitle.setText(selectedMovie.name().getValue());
```
Comment (TomassSimko) [inline src/main/java/com/movie_collection/gui/controllers/MovieController.java:114]: Here is duplicate with title but I also fixed it in pr

File: src/main/java/com/movie_collection/gui/controllers/MovieController.java [hunk 3]
```diff
@@ -264,7 +262,7 @@ private void showMediaPlayerUnselected() throws SQLException {
         Alert alert = new Alert(Alert.AlertType.INFORMATION);
         alert.setHeaderText("Select your Media Player");
         alert.getButtonTypes().setAll(new ButtonType("OK"));
-        Optional<ButtonType> btn = alert.showAndWait();
+        alert.showAndWait();
```
Comment (TomassSimko) [inline src/main/java/com/movie_collection/gui/controllers/MovieController.java:265]: I also fixed this so no problem :)

**Review by TomassSimko — COMMENTED** [review]
If u want change it otherwise as we talked when its gonna be merge conf we just take it in in the comments otherwise sweet 🙏

**Comment by TomassSimko** [comment]
Do you want us to approve this so u can continue with css ?

**Comment by patrikvalentiny** [comment]
Up to you guys, I assumed it would be better to have these changes in prod so we are working on same base

**Review by TomassSimko — APPROVED** [review]
There u go :)

**Comment by TomassSimko** [comment]
We just need @LordStacker ✋ for the rest of pr as well :)

**Review by LordStacker — APPROVED** [review]

**Pull request merged**